packages/augur-core/src/contracts/gsn/v2/BatchForwarder.sol

```diff
@@ -2,15 +2,21 @@
 pragma solidity 0.5.15;
 pragma experimental ABIEncoderV2;
 
-import "ROOT/gsn/v2/TrustedForwarder.sol";
+import "ROOT/gsn/v2/Forwarder.sol";
 import "ROOT/gsn/v2/BaseRelayRecipient.sol";
 import "ROOT/gsn/v2/utils/GsnUtils.sol";
 
 /**
  * batch forwarder support calling a method sendBatch in the forwarder itself.
  * NOTE: the "target" of the request should be the BatcherForwarder itself
  */
-contract TrustedBatchForwarder is TrustedForwarder, BaseRelayRecipient {
+contract BatchForwarder is Forwarder, BaseRelayRecipient {
+
+    function versionForwarder() external view returns (string memory){
+        return "2.0.0-alpha.1+opengsn.batched.iforwarder";
```

Comment:
are we going to be stuck with this `version` text? What about when version 2.0.0 comes out for real?

Reply:
yea its just to satisfy an interface